Dockerfile

```diff
@@ -1,7 +1,12 @@
-FROM node:10
+FROM metabrainz/node:10 as bookbrainz-base
+
+ARG DEPLOY_ENV
 
 ARG BUILD_DEPS=" \
-    build-essential"
+    build-essential \
+    git \
+    python-dev \
+    libpq-dev"
 
 ARG RUN_DEPS=" \
```

Comment:
This can be removed.
In the `RUN` command below, replace `curl -sL https://deb.nodesource.com/setup_10.x | bash - && \` with `apt-get update && \`  and remove the reference to `$RUN_DEPS`

At the moment, I think it may be reinstalling node :p